src/scss/components/optional/_cookie-warning.scss

```diff
@@ -45,6 +45,7 @@
 		background: url('//dy3erx8o0a6nh.cloudfront.net/images/icon-close-banner.png') no-repeat center center;
 		background-size: 10px 10px;
 		border: none;
+		cursor: pointer;
```

Comment:
Should this go on the `: hover` selector below?

Reply:
As cursor only applies on-hover they're functionally equivalent, but I can see your argument from a stylistic point of view

Reply:
Agree with this one – should be applied on `:hover` (as would ensure no collisions should styles ever get applied in other states (like `:disabled`).

May not be an issue, but would def make sure it isn't.